.github/workflows/commit-checker.yml

```diff
@@ -0,0 +1,90 @@
+# Manually run: commit checker
+
+name: commit-checker
+
+# Triggered on push to master,
+# or PR against master.
+on:
+  workflow_dispatch:
+    inputs:
+      end-ref:
+        description: CLDR Commit to check
+        required: true
+        default: 'master'
+      fix-version:
+        description: Fix Version (e.g. 38)
+        default: 38 # TODO: remove, for testing only
```

Comment:
When can/should these defaults be changed or removed?

Reply:
What I thought of doing is to leave the defaults in for now to make it easier to test (you can just click and click).   And then when we change the DNS over, then remove those so that you *have* to put in a version number.  There's no reason to keep updating the defaults then.

Reply:
Also, right now it points at my branch… while we are still making changes to the tool. when CLDR is happy with the output, then i will attempt to merge my branch into ICU. Then I can update this workflow to point at ICU proper.

Reply:
I've removed many of the defaults.